packages/appstore/src/components/wallet-content/wallet-option-multipliers-listing.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Text, StaticUrl } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import ListingContainer from 'Components/containers/listing-container';
+import { BrandConfig } from 'Constants/platform-config';
+import TradingAppCard from 'Components/containers/trading-app-card';
+import PlatformLoader from 'Components/pre-loader/platform-loader';
+import { getHasDivider } from 'Constants/utils';
+import { TCoreStores } from '@deriv/stores/types';
+import { useStore, observer } from '@deriv/stores';
+
+type TProps = {
+    account: TCoreStores['client']['accounts'][0];
+};
+
+const WalletOptionsAndMultipliersListing = observer(({ account }: TProps) => {
+    const { traders_hub, client, ui } = useStore();
+    const { is_mobile } = ui;
+    const { is_landing_company_loaded } = client;
+    const { available_platforms } = traders_hub;
+
+    const { is_virtual, landing_company_shortcode } = account;
+    const is_eu = landing_company_shortcode === 'maltainvest' || landing_company_shortcode === 'malta';
```

Comment:
try to use content_flag which we have in the `traders-hub-store`

Reply:
@mahdiyeh-deriv about `content_flag` in `traders-hub-store`, I thought about this, but this property just for current account. For example, there is DIEL client which has SVG and MALTA wallets at the same time, in this case I can't take this property from `traders-hub-store` because it's related to only the current account. In my opinion this information `is_eu` can be different for different wallets for one user. Let me know what do you think?

Reply:
This flag is based on the landing_company of the user which will not be changed for wallets as well. I get your concern here, in this case you need combination of flags and not one of them.
checkout `is_demo_low_risk` observable in the traders-hub-store, you can make a hook based on the different conditions you need and use it all around the code instead of adding conditions in each file.

Reply:
@mahdiyeh-deriv Thanks for your reply, I'll think a little bit more what can I do with it. But this will be actual when API start to work properly, so in my opinion it's out of the scope of the card right now. Anyway I'll see what I can do.

Reply:
I agree with @mahdiyeh-deriv. I think its better to extract this to a hook, so then other devs can reuse your hook.